96.md

```diff
@@ -84,7 +84,8 @@ See https://github.com/aljazceru/awesome-nostr#nip-96-file-storage-servers.
 
 ## Auth
 
-When indicated, `clients` must add an [NIP-98](98.md) `Authorization` header (**optionally** with the encoded `payload` tag set to the base64-encoded 256-bit SHA-256 hash of the file - not the hash of the whole request body).
+When indicated, `clients` must add an [NIP-98](98.md) `Authorization` header.
+`clients` SHOULD include either the `payload` tag or the `payload_multipart` tag with the `file` multipart field name included in the digest (`["payload_multipart", "<sha256-of-file>", "file"]`).
```

Comment:
```suggestion
When indicated, `clients` must add an [NIP-98](98.md) `Authorization` header (or form field if using an html form). It may include the `payload_multipart` tag with just the `file` field hash.
```

Considering the `payload` tag was optional, it is safe to use just the new `payload_multipart` one.

Reply:
I think I agree. My reasoning for allowing either was basically backwards compatibility, but there's really nothing to be backwards compatible with afaict.